packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts

```diff
@@ -41,8 +41,8 @@ const source = "bicep-envchecker";
 export async function ensureBicep(ctx: SolutionContext): Promise<string> {
   const bicepChecker = new BicepChecker(ctx.logProvider, ctx.telemetryReporter);
   try {
-    if (bicepChecker.isEnabled() && !bicepChecker.isInstalled()) {
-      bicepChecker.install();
+    if ((await bicepChecker.isEnabled()) && !(await bicepChecker.isInstalled())) {
+      await bicepChecker.install();
```

Comment:
Should we notify user that we will install bicep binary for them or ask them for permission?

Reply:
We notify user by logging like `Dotnet SDK` and `Azure Function Core Tools.
There's a popup with a help link if fail.
It's not a global install, so we don't ask user for permission.(install dir: ~/.fx/bin/bicep)